invenio_drafts_resources/services/draft_file_metadata_service.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
```

Comment:
moderate: same file naming issue than in records-resources

Reply:
Done